.github/dependabot.yml

```diff
@@ -5,6 +5,6 @@ updates:
   schedule:
     interval: monthly
   open-pull-requests-limit: 10
-  ignore_updates:
+  ignored_updates:
```

Comment:
I don't see any description for either `ignore_updates` or `ignored_updates` in the [dependabot docs](https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates).  The Jenkins core ignore file uses `ignore` as do the plugins where I've seen it used.